src/components/PostForm.jsx

```diff
@@ -1,37 +1,86 @@
 import React, { useState } from 'react'
-import { TextInput, Button, FlatList, SafeAreaView } from 'react-native'
-import { ListItem } from 'react-native-elements'
+import { TextInput, Button, FlatList, SafeAreaView, Text } from 'react-native'
+import { ListItem, Card } from 'react-native-elements'
 import TrackService from '../modules/TrackService'
+import TrackPlayer from './TrackPlayer'
 import { useSelector } from 'react-redux'
+import store from '../state/store/store'
+import styles from '../styles/styles'
+import PostService from "../modules/PostService";
 
-const PostForm = () => {
+const PostForm = (props) => {
   const [search, setSearch] = useState()
-  const { searchResult } = useSelector(state => state)
+  const [description, setDescription] = useState()
+  const { searchResult, trackDetails, errorMessage } = useSelector((state) => state)
 
   return (
     <SafeAreaView>
       <TextInput
+        style={styles.searchInput}
         testID='searchInput'
-        placeholder='Search here...'
-        onChangeText={text => setSearch(text)}
+        placeholder='Search a song to create a post!'
+        onChangeText={(text) => setSearch(text)}
         value={search || ''}
       />
       <Button
+        style={styles.searchButton}
         testID='searchButton'
         title='Search'
+        color='black'
```

Comment:
you should be able to change the color of the button in the styles component `searchButton`

Reply:
We tried putting `color` and `background color` in the style sheet, but both didn't work unfortunately...